src/main/java/org/spectrumauctions/sats/core/bidlang/generic/SimpleRandomOrder/XORQRandomOrderSimple.java

```diff
@@ -77,7 +75,7 @@ public void setIterations(int iterations) {
      */
     @Override
     public Iterator<GenericValue<T, S>> iterator() {
-        return new SimpleRandomOrderIterator(iterations, rngSupplier.getUniformDistributionRNG(seed));
+        return new SimpleRandomOrderIterator(iterations, rngSupplier.getUniformDistributionRNG());
```

Comment:
Like this, if you call 'iterator()' multiple times you get different orders every time, right? Is this intended?
Generally asked: Can we ensure reproducibility like this?